runtime/stdlib.js

```diff
@@ -131,7 +131,16 @@ var caml_named_values = {};
 //Provides: caml_register_named_value (const,mutable)
 //Requires: caml_named_values, caml_jsbytes_of_string
 function caml_register_named_value(nm,v) {
-  caml_named_values[caml_jsbytes_of_string(nm)] = v;
+  nm = caml_jsbytes_of_string(nm);
+  if (!caml_named_values.hasOwnProperty(nm)) caml_named_values[nm] = v;
```

Comment:
Why such change ? This is not what the upstream runtime does https://github.com/ocaml/ocaml/blob/48d3c5a065838e41388b18ea8f33d57703ef4337/runtime/callback.c#L403

Reply:
I don't remember why I did that. Maybe I misread the C code? I think we can remove this change.
I will need to fix the Wasm runtime as well.